pages/coinbase/signin.py

```diff
@@ -40,61 +41,63 @@ def __init__(self, driver, spawn=False, bypass_auth=True):
 
     # NAVIGATION BAR
 
-    home_link_loc =     (by.CLASS_NAME, 'Header__Logo')
```

Comment:
your indentation is inconsistent.

Tip: don't change other people's indentation that has already been approved for merge.... you're only changing something that has already been accepted and opening it up for rejection. Just add your own things, only edit if you have to.

Reply:
I am trying to match all indentations. Which is why I adjusted those